racingcar-java/src/test/java/calculator/CalculatorTest.java

```diff
@@ -0,0 +1,80 @@
+package calculator;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.regex.Pattern;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+class CalculatorTest {
+
+    private Calculator calculator = new Calculator();
+
+    @Test
+    @DisplayName("덧셈")
+    void 문자열계산기_덧셈() {
+        assertThat(calculator.calculator("2 + 3")).isEqualTo(5);
+    }
+
+    @Test
+    @DisplayName("뺄셈")
+    void 문자열계산기_뺄셈() {
+        assertThat(calculator.calculator("3 - 1")).isEqualTo(2);
+    }
+
+    @Test
+    @DisplayName("곱셈")
+    void 문자열계산기_곱셈() {
+        assertThat(calculator.calculator("3 * 2")).isEqualTo(6);
+    }
+
+    @Test
+    @DisplayName("나눗셈")
+    void 문자열계산기_나눗셈() {
+        assertThat(calculator.calculator("9 / 3")).isEqualTo(3);
+    }
+
+    @Test
+    @DisplayName("null")
+    void 문자열계산기_null() {
+        assertThatExceptionOfType(IllegalArgumentException.class)
+                .isThrownBy(() ->
+                        calculator.calculator(null)
+                ).withMessageMatching("입력값이 null 임");
```

Comment:
```suggestion
                .isThrownBy(() -> calculator.calculator(null))
                .withMessageMatching("입력값이 null 임");
```
람다식은 굳이 한칸 띄우지 않습니다 :)